app/scripts/convert.py

```diff
@@ -47,7 +48,23 @@ def to_keys_indexes_str(_input):
 
 def convert(file_path):
     with Path(file_path).open("r") as input_file:
-        input_data = json.load(input_file)
+        csv_data = csv.reader(input_file, delimiter=",")
+        rows_data = []
+
+        headers = []
+        headers_row = 1
+        for row in csv_data:
+            if headers_row == 1:
+                headers = row
+                headers_row = 0
+            else:
+                row_data = {}
+                for i in range(0, len(row)):
+                    if row[i]:
```

Comment:
This looks to be the condition that incorrectly skips empty fields. Is this solving another problem?

Reply:
I see what it's doing. If `id` is blank, it doesn't include it and allows django to generate it.

Maybe the right way to resolve this is to say something like all fields must be populated.